.github/workflows/release.yml

```diff
@@ -1,36 +1,20 @@
+name: Release sonar-dummy-js
```

Comment:
just name it `release` (same as what we have for other repositories https://github.com/SonarSource/sonar-dummy/blob/master/.github/workflows/release.yml#L1 #coherence